Plugins/Json/src/AmbiguityConfigJsonConverter.cpp

```diff
@@ -35,15 +35,26 @@ void from_json(const nlohmann::json& j, ConfigPair& p) {
 
     detectorConfig.sharedHitsFlag = value["sharedHitsFlag"];
 
-    std::vector<double> factorHits = value["factorHits"];
-    std::vector<double> factorHoles = value["factorHoles"];
+    std::vector<double> goodHits = value["goodHits"];
+    std::vector<double> goodHoles = value["goodHoles"];
 
-    for (auto factor : factorHits) {
-      detectorConfig.factorHits.push_back(factor);
+    std::vector<double> fakeHits = value["fakeHits"];
+    std::vector<double> fakeHoles = value["fakeHoles"];
```

Comment:
why assign them every single time? allegedly they are always the same numbers, right? Also, why a copy?

Reply:
So you think its better to have a fixed config in a header file in athena than use the from json file to load it every time ? 
The core algorithm doesn't care how the detector config is assigned. So from_json is in the sense an optional tool.

Reply:
my point was that this assignment is done inside a loop. But I assume that `value["fakeHits"]` remains the same for every iteration. Right?

Reply:
ah no sorry... I misunderstood the code. Forget about this point